### PR TITLE
Add Engagment.description

### PR DESCRIPTION
**Pull request by andrewmurraydavid**
Added summary field to the engagement object

┆Issue is synchronized with this [Monday item](https://seed-company-squad.monday.com/boards/3451697530/pulses/3629667485) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by CarsonF — COMMENTED** [review]
Can you call it `description` instead? I think that makes more sense.

File: src/components/engagement/dto/update-engagement.dto.ts [hunk 1]
```diff
@@ -33,6 +40,9 @@ export abstract class UpdateEngagement {
 
   @Field(() => EngagementStatus, { nullable: true })
   readonly status?: EngagementStatus;
+
+  @RichTextField({ nullable: true })
+  readonly summary?: RichTextDocument;
```
Comment (CarsonF) [inline src/components/engagement/dto/update-engagement.dto.ts:45]:
```suggestion
  readonly summary?: RichTextDocument | null;
```

File: src/components/engagement/dto/update-engagement.dto.ts [hunk 2]
```diff
@@ -2,7 +2,14 @@ import { Field, InputType, ObjectType } from '@nestjs/graphql';
...
 import { stripIndent } from 'common-tags';
-import { CalendarDate, DateField, ID, IdField } from '../../../common';
+import { RichTextField } from '~/common/rich-text.scalar';
+import {
+  CalendarDate,
+  DateField,
+  ID,
+  IdField,
+  RichTextDocument,
+} from '../../../common';
```
Comment (CarsonF) [inline src/components/engagement/dto/update-engagement.dto.ts:12]:
```suggestion
import {
  CalendarDate,
  DateField,
  ID,
  IdField,
  RichTextDocument,
  RichTextField,
} from '~/common';
```

**Comment by andrewmurraydavid** [comment]
Sure, I just used what was provided in the ticket

File: src/components/engagement/dto/update-engagement.dto.ts (hunk 2 above)
Comment (andrewmurraydavid) [reply src/components/engagement/dto/update-engagement.dto.ts:12]: Yup, I'll get to that in the morning after the rename

**Comment by CarsonF** [comment]
> Sure, I just used what was provided in the ticket

Tried to call it out before it got to you. https://github.com/SeedCompany/cord-field/issues/1333#issuecomment-1337916596
Np sleep well.

**Review by CarsonF — APPROVED** [review]
Thanks @andrewmurraydavid! Looks great.

**Pull request merged**